src/utils/helpers/structure/addChangeRequestForReviewer.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  ReviewResponse,
+  ReviewResponseDecision,
+  ReviewStatus,
+  TemplateElementCategory,
+} from '../../generated/graphql'
+import { FullStructure } from '../../types'
+
+const hasConsolidatorReviewResponse = (reviewerReviewResponse?: ReviewResponse) =>
+  (reviewerReviewResponse?.reviewResponsesByReviewResponseLinkId?.nodes || []).length > 0
+
+const addChangeRequestForReviewer = (structure: FullStructure) => {
+  const questionElements = Object.values(structure?.elementsById || {}).filter(
+    ({ element }) => element?.category === TemplateElementCategory.Question
+  )
+
+  const hasConsolidatorReviewResponses = questionElements.some(
```

Comment:
this methods looks great, one suggestion (not required change at all), would be to 'filter' for consolidatorReviewResponses here and us the in the forEach below

Reply:
Ah, I think the last lines of the `forEach` block are meant to update **every** reviewResponse with `isChangeRequest` and `isChanged` fields, so we don't have undefined ones. This is to simplify other areas of the code.